### PR TITLE
contributors: add benchmark guide

### PR DESCRIPTION
**Pull request by SunRunAway**

### CODE REVIEW

**Comment by SunRunAway** [comment]
ping.

File: contributors/README.md [hunk 1]
```diff
@@ -167,6 +168,26 @@ TBD
...
+### Benchmark
+
+#### TiDB
+
+It's appreciate to write benchmark code and add benchmark data if the change affects performance. The [benchstat](https://godoc.org/golang.org/x/perf/cmd/benchstat) tool is conventionally used to format benchmark data for change descriptions.
+
+We could do is run the benchmarks over multiple runs ie
+
+```bash
+go test -run=^$ -bench=BenchmarkXXX -count=10
```
Comment (zz-jason) [inline contributors/README.md]: which directory should we enter in order to run this command successfully?

Comment (SunRunAway) [reply contributors/README.md]: I think it's a common knowledge for go user.

File: contributors/README.md [hunk 2]
```diff
@@ -167,6 +168,26 @@ TBD
 
 TBD
 
+### Benchmark
+
+#### TiDB
```
Comment (dcalvin) [inline contributors/README.md]:
```suggestion
```

File: contributors/README.md [hunk 3]
```diff
@@ -167,6 +168,26 @@ TBD
...
+
+It's appreciate to write benchmark code and add benchmark data if the change affects performance. The [benchstat](https://godoc.org/golang.org/x/perf/cmd/benchstat) tool is conventionally used to format benchmark data for change descriptions.
```
Comment (dcalvin) [inline contributors/README.md]:
```suggestion
If the change affects TIDB's performance, the benchmark data is normally required in the description. You can use the [benchstat](https://godoc.org/golang.org/x/perf/cmd/benchstat) tool to format benchmark data for change descriptions.
```

File: contributors/README.md [hunk 4]
```diff
@@ -167,6 +168,25 @@ TBD
 
 TBD
 
+### Benchmark
+
+
+If the change affects TIDB's performance, the benchmark data is normally required in the description. You can use the [benchstat](https://godoc.org/golang.org/x/perf/cmd/benchstat) tool to format benchmark data for change descriptions.
+
+We could do is run the benchmarks over multiple runs ie
```
Comment (dcalvin) [inline contributors/README.md]:
```suggestion
The following script runs benchmark multiple times (10)
```

File: contributors/README.md [hunk 5]
```diff
@@ -167,6 +168,25 @@ TBD
...
+
+```bash
+go test -run=^$ -bench=BenchmarkXXX -count=10
+```
+
+And we could make more clear and reportable differences by copying and pasting that output to two different files: old.txt and new.txt and then running:
```
Comment (dcalvin) [inline contributors/README.md]:
```suggestion
**Tip**: To make the result more readable, you can copy and paste the output to two both the old.txt and new.txt, and then run:

File: contributors/README.md [hunk 6]
```diff
@@ -167,6 +168,25 @@ TBD
...
+
+If the change affects TIDB's performance, the benchmark data is normally required in the description. You can use the [benchstat](https://godoc.org/golang.org/x/perf/cmd/benchstat) tool to format benchmark data for change descriptions.
+
+The following script runs benchmark multiple times (10)
+
+```bash
+go test -run=^$ -bench=BenchmarkXXX -count=10
+```
+
+**Tip**: To make the result more readable, you can copy and paste the output to two both the old.txt and new.txt, and then run:
```
Comment (dcalvin) [inline contributors/README.md]:
```suggestion
**Tip**: To make the result more readable, you can copy and paste the output to both the old.txt and new.txt, and then run: 
```

File: contributors/README.md [hunk 7]
```diff
@@ -167,6 +168,25 @@ TBD
...
+go test -run=^$ -bench=BenchmarkXXX -count=10
+```
+
+**Tip**: To make the result more readable, you can copy and paste the output to two both the old.txt and new.txt, and then run: 
+
+```
+benchstat old.txt new.txt
+```
+
+and paste the result into your pull request description. Here is a good [example](https://github.com/pingcap/tidb/pull/12903#issue-331440170).
```
Comment (dcalvin) [inline contributors/README.md]:
```suggestion

Paste the result into your pull request description. Here is a good [example](https://github.com/pingcap/tidb/pull/12903#issue-331440170).
```

**Pull request merged**